packages/react/src/components/Authenticator/VerifyUser/VerifyUser.tsx

```diff
@@ -29,17 +29,24 @@ const {
 const generateRadioGroup = (
   attributes: UnverifiedUserAttributes
 ): JSX.Element[] => {
-  return Object.entries(attributes).map(([key, value]: [string, string]) => {
-    const verificationType = (
-      defaultFormFieldOptions[key] as { label: ContactMethod }
-    ).label;
-    return (
-      <Radio name="unverifiedAttr" value={key} key={key}>
-        {translate(verificationType)}:{' '}
-        {censorContactMethod(verificationType, value)}
-      </Radio>
-    );
-  });
+  return Object.entries(attributes).map(
+    ([key, value]: [string, string], index) => {
+      const verificationType = (
+        defaultFormFieldOptions[key] as { label: ContactMethod }
+      ).label;
+      return (
+        <Radio
+          name="unverifiedAttr"
+          value={key}
+          key={key}
+          {...(index === 0 ? { checked: true } : null)}
```

Comment:
Open to feedback on this, but Wwhen I tried to use `checked={index === 0}`, I wasn't able to select the second radio button from the UI. It looked like it was re-rendering on each click and would reset the selection to the first radio button. 

Doing it this way, I was able to choose the other radio button, but maybe I'm missing something?

Reply:
I think for radio buttons `checked` is supposed to represent the currently selected value, not the default. I tried with `defaultChecked={index===0}` and it worked. I'm pretty sure that's just a react thing 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#checked

Reply:
I also noticed the RadioGroupField has a prop for defaultValue, not sure if that's helpful here - https://ui.docs.amplify.aws/react/components/radiogroupfield#usage

Reply:
Thank you @esauerbo! I didn't know about the `defaultChecked`. I'll change it to that

Thanks @thaddmt. I did look into that, but I think for it to work I would have to know the full name of the radio button, like `email: ******@email.com` or something like that, depending on what was being used. I thought it would've been more straightforward to just check it than trying to find the string